zenpy/lib/endpoint.py

```diff
@@ -293,7 +293,7 @@ def __call__(self, query=None, **kwargs):
                     value = '"{}"'.format(value)
                 renamed_kwargs.update({key + ':': '%s' % value})
 
-        search_query = ['%s%s' % (key, value) for (key, value) in renamed_kwargs.items()]
+        search_query = ['%s%s' % (k, v) for (k, v) in renamed_kwargs.items()]
```

Comment:
This rename was required by python 27 checks. It raised some strange pep8 error with value reassignment from line 293.